README.md

```diff
@@ -19,13 +19,15 @@ An overview of the advantages and differences as compared to the TF wrappers can
 This package is available on PyPi:
 
 ```bash
-pip install scikeras
+# Normal tensorflow
+pip install scikeras[tensorflow]
+
+# or tensorflow-cpu
+pip install scikeras[tensorflow-cpu]
 ```
 
 The current version of SciKeras depends on `scikit-learn>=1.0.0` and `TensorFlow>=2.7.0`.
 
-You will need to manually install TensorFlow; due to TensorFlow's packaging it is not a direct dependency of SciKeras.
-You can do this by running:
```

Comment:
I think it's worth leaving a comment in explaining why we recommend installing with extras. Something like:

> SciKeras packages TensorFlow as an optional dependency because there are several flavors of TensorFlow available (`tensorflow`, `tensorflow-cpu`, etc.). `pip install scikeras[tensorflow]` is equivalent to `pip install scikeras tensorflow` and is offered just for convenience. You can also install just SciKeras with `pip install scikeras`, but you will need a version of tensorflow installed at runtime or SciKeras will throw an error when you try to import it.